ksql-common/src/test/java/io/confluent/ksql/util/SchemaUtilTest.java

```diff
@@ -316,23 +316,17 @@ public void shouldCreateCorrectSchemaDescription() {
         .field("mapcol", schemaBuilder.map(Schema.STRING_SCHEMA, Schema.FLOAT64_SCHEMA))
         .field("address", addressSchema).build();
 
-    String schemaDescription = SchemaUtil.describeSchema(structSchema);
-
-    assertThat(schemaDescription, equalTo("STRUCT < \n"
-                                          + "\t ordertime BIGINT, \n"
-                                          + "\t orderid BIGINT, \n"
-                                          + "\t itemid VARCHAR(STRING), \n"
-                                          + "\t orderunits DOUBLE, \n"
-                                          + "\t arraycol ARRAY[DOUBLE], \n"
-                                          + "\t mapcol MAP[VARCHAR(STRING),DOUBLE], \n"
-                                          + "\t address STRUCT < \n"
-                                          + "\t NUMBER BIGINT, \n"
-                                          + "\t STREET VARCHAR(STRING), \n"
-                                          + "\t CITY VARCHAR(STRING), \n"
-                                          + "\t STATE VARCHAR(STRING), \n"
-                                          + "\t ZIPCODE BIGINT\n"
-                                          + " >\n"
-                                          + " >"));
+    /*
```

Comment:
Why is this commented out?

Reply:
The function under test was removed. Thought I got rid of the test too.